docs/conf.py

```diff
@@ -273,7 +273,7 @@ def check_sphinx_version(expected_version):
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-html_theme_options = {"collapse_navigation": True, "display_version": True}
+html_theme_options = {"collapse_navigation": True}
```

Comment:
```suggestion
html_theme_options = {"collapse_navigation": True, "version_selector": True}
```
I'm not sure if this works but it is one of the recommended replacements for `display_version`.